lib/ddtrace/contrib/qless/qless_job.rb

```diff
@@ -52,10 +52,7 @@ def after_fork
           # Add a pin, marking the job as forked.
           # Used to trigger shutdown in forks for performance reasons.
           # Cleanup happens in the TracerCleaner class
-          Datadog::Pin.new(
-            configuration[:service_name],
-            config: { forked: true }
-          ).onto(::Qless)
+          Datadog::Tracing.configure_onto(::Qless, forked: true)
```

Comment:
Just to confirm: why don't we need to pass the service name to the pin anymore? Is it because the default will be used anyway, so we don't need to copy it?